src/drafter.cc

```diff
@@ -110,18 +123,36 @@ namespace { // FIXME: cut'n'paste from main.cc - duplicity
 }
 
 /* Serialize result to given format*/
-DRAFTER_API char* drafter_serialize(drafter_result *res, const drafter_options options) {
+DRAFTER_API char* drafter_serialize(drafter_result *res, const drafter_serialize_options serialize_opts) {
+
+    drafter::SerializeFormat format = drafter::UnknownFormat;
 
     if (!res) {
         return nullptr;
     }
 
-    drafter::WrapperOptions wrapperOptions(options.sourcemap);
+    switch (serialize_opts.format)
+    {
+        case DRAFTER_SERIALIZE_JSON:
+            format = drafter::JSONFormat;
+            break;
+
+        case DRAFTER_SERIALIZE_YAML:
+            format = drafter::YAMLFormat;
+            break;
+
+        default:
+            return nullptr;
+    }
+
+    drafter::WrapperOptions wrapperOptions(serialize_opts.sourcemap);
```

Comment:
I am okay with that. I am just trying to think if removing wrapperOptions later will mess things up or not. Can you take a look over it and make sure we are okay with not removing it now?